images/scripts/fedora-43-anaconda-payload.bootstrap

```diff
@@ -3,4 +3,9 @@ set -eux
 
 OUTPUT="$1"
 
-PYTHONPATH=. python3 images/scripts/create-anaconda-payload --output=$OUTPUT --image fedora-43
+WORKDIR=$(mktemp -d)
+PYTHONPATH=. python3 images/scripts/create-anaconda-liveimg-payload --output="$WORKDIR" --image fedora-43
+PYTHONPATH=. python3 images/scripts/create-anaconda-dnf-payload --output="$WORKDIR" --image fedora-43
```

Comment:
Each of these scripts does a new reboot/shutdown, which slows down the build. Not a biggie, just a bit of an inconvenience.